cmd/applications-rp/applications-rp-dev.yaml

```diff
@@ -49,7 +50,8 @@ logging:
   json: false
 # Tracing configuration
 tracerProvider:
-  serviceName: "applications.core"
+  enabled: false
+  serviceName: applications-rp
```

Comment:
This was `applications.core` before it's not really right. Changed to `applications-rp`.